install/resources/settings.xml

```diff
@@ -959,7 +959,8 @@ disabled=Disable this feature]]></optionscode>
 		</setting>
 		<setting name="username_method">
 			<title>Allowed Login Methods</title>
-			<description><![CDATA[The login methods you wish to allow for the username field. Username only, Email only, or allow both.]]></description>
+			<description><![CDATA[The login methods you wish to allow for the username field. Username only, Email only, or allow both.<br />
+<strong>Note:</strong> This will only work if there are no duplicate accounts using the same e-mail address.]]></description>
```

Comment:
`This` should rather be `Email options` to be more clear, because changing to `Username only` will always work fine.